packages/icons/scripts/buildSvg.js

```diff
@@ -17,7 +17,7 @@ const normalizeAndCopySvg = (srcDir, distDir) => {
         const svgPath = path.join(srcDir, svg);
         const svgDistPath = path.join(distDir, svg);
         const svgContent = fs.readFileSync(svgPath, 'utf8');
-        const svgContentFixed = svgContent.replace(/fill="#\w+"/g, 'fill="currentColor"');
+        const svgContentFixed = svg.endsWith('-colored.svg') ? svgContent : svgContent.replace(/fill="#\w+"/g, 'fill="currentColor"');
```

Comment:
I strongly suggest creating a function for this -> `const normalizeSvgColors = (svg, svgContent) => svg.endsWith('-colored.svg') ? svgContent : svgContent.replace(/fill="#\w+"/g, 'fill="currentColor"');`

`const svgContentNormalized = normalizeSvgColors(svg, svgContent)`.

In this case the `svgContentFixed` name is out-of-date ;-)

Reply:
done